src/DacpacTool/PackageBuilder.cs

```diff
@@ -388,5 +388,13 @@ private List<int> ParseSuppressionList(string suppressionList)
 
             return result;
         }
+
+        public void GenerateCreateScript(FileInfo dacpacFile, string databaseName)
```

Comment:
Hmm, I was just thinking we might want to write something to the console here so that we can see that this is actually happening?